EnglishTranslation-master/SKILL.tsv

```diff
@@ -2026,7 +2026,7 @@ SKILL_20150317_002025	Spear Blow has a chance to stun.{nl} (Increases by 5% per
 SKILL_20150317_002026	Surprise Attack	기습				
 SKILL_20150317_002027	Additional critical chance against stunned enemies. {nl} (Basic chance of 30% + 5% added per level.)	스턴 상태인 적 공격시 일정확률로 추가 데미지를 줍니다.{nl}(기본 30% + 레벨당 5%씩 추가)				
 SKILL_20150317_002028	Concentration	집중				
-SKILL_20150317_002029	Critical rate increases when attacking far away enemies. {nl} (Basic chance of 50% + 5% added per level.)	최대 사거리 근처에 있는 적 공격시 일정확률로 치명타가 들어갑니다.{nl}(기본 50% + 레벨당 5%씩 추가)				
+SKILL_20150317_002029	Critical Rate increases when attacking far away enemies. {nl} (Basic chance of 50% + 5% added per level.)	최대 사거리 근처에 있는 적 공격시 일정확률로 치명타가 들어갑니다.{nl}(기본 50% + 레벨당 5%씩 추가)				
```

Comment:
I am not sure if IMC has kept the standardization for these kind of lines, but I prefer it like this:

`Increases your Critical Rate when attacking far away enemies` 

Mention 'Increase/Increases' first. Take a look at the R1~R8 attributes and you see many of them are like this.